lib/marc_work.rb

```diff
@@ -69,4 +69,8 @@ def get_link_status
     return status
   end
 
+  def reset_to_new
+    first_occurance("001").content = "__TEMP__"
```

Comment:
```suggestion
    first_occurence("001").content = "__TEMP__"
```

Reply:
Although since this is a function call it probably needs to be changed elsewhere as well?

Reply:
It happens often, yes:
$ git grep first_occurance | wc -l
164